indigo_app/templates/indigo_api/work_layout.html

```diff
@@ -21,13 +21,14 @@
       {% block main-header-title %}
         {% if work.pk %}
           <h5 class="main-header-title">
-            <span class="work-title">{% if work.pk %}{{ work.title }}{% else %}Create a new work{% endif %}</span> 
```

Comment:
I think this is necessary for the new work page?

Reply:
the whole block is wrapped in `if work.pk`?

Reply:
Currently I think it gives `(untitled work)` here for a new work

Reply:
Ah, you make a strong argument!